sdk/sqlmanagement/Azure.ResourceManager.Sql/src/Azure.ResourceManager.Sql.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Version>1.0.0-preview.1</Version>
+    <PackageId>Azure.ResourceManager.Sql</PackageId>
+    <Description>Azure Resource Manager client SDK for Azure resource provider Microsoft.Sql</Description>
+    <PackageTags>azure;management;arm;resource manager;sql</PackageTags>
```

Comment:
Where do we represent the underlying api-version for this package?

Reply:
Tracked in AB#4094